azext_iot/tests/test_constants.py

```diff
@@ -4,6 +4,17 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from enum import Enum
+
+
+class ResourceTypes(Enum):
+    """
+    Resource types to use with az resource
+    """
+    central = "Microsoft.IoTCentral/IoTApps"
```

Comment:
You can also use the fully qualified resource Id which already has the resource type.

Reply:
I was going to use that but then I needed an extra call to get the resource id of existing resources (you can get the id from any create call if the resource is being generated but if the resource exists, we do not do those calls/check if the resource exists...) 

I figured since all test classes have the name + rg and the types _should_ not change, this would be a suitable alternative. Unless you think we should have a show call when the resource is provided.

Reply:
Ok, the reasoning makes sense, I'm fine with keeping this.